pkg/controller/certificates/approver/sarapprove.go

```diff
@@ -100,7 +100,7 @@ func (a *sarApprover) handle(csr *capi.CertificateSigningRequest) error {
 		}
 		if approved {
 			appendApprovalCondition(csr, r.successMessage)
-			_, err = a.client.CertificatesV1beta1().CertificateSigningRequests().UpdateApproval(csr)
+			_, err = a.client.CertificatesV1beta1().CertificateSigningRequests().UpdateApproval(context.Background(), csr, metav1.UpdateOptions{})
```

Comment:
Background?

Reply:
I saw us using background context in other async paths like controllers. It's functionally equivalent to TODO, happy to change it to TODO in a follow up if you think background is wrong here